data/json/recipes/recipe_food.json

```diff
@@ -3049,6 +3056,75 @@
   ]
 },{
   "type" : "recipe",
+  "result": "brew_milk_curdled",
+  "result_mult" : 3,
```

Comment:
Fixed it. 15 units of milk now create 18 units of curdled milk, which create 9 portions of cheese.